codehs/scratchjr
# ScratchJR Fullscreen Fix [100%]

Description:
Fixes auto-fullscreen mode on scratch jr demos

@willm0602 @almeidaalajoel 

Features:
- [x] Fixes fullscreen on scratch jr demos

Issues:
- fixes ENG-5436

Screenshots:
https://www.loom.com/share/7e325eb26646480ab1a717327e305f3c?sid=11d12f96-a667-4c0f-b280-ba4dbeae7ae1

Steps to Test:
Create a scratch jr demo item. (If done in the shell, make sure to make a STARTER_CODE SavedBlob along with it and attach it to the item)
Go to the demo editor page for that item. (http://localhost:8000/elementary/scratchjr/demo/ITEM_ID)
(Make sure your scratch jr is bundled) See that the page loads and goes into fullscreen automatically.

Staging URLs:
- https://staging.codehs.com/elementary/scratchjr/demo/12899763

Live URLs:
- https://codehs.com/elementary/scratchjr/demo/12899763
